### ISSUE TITLE
Puppetting webrecorder-desktop

### ISSUE THREAD

**Issue by jswrenn** [webrecorder/webrecorder-desktop#43]
Is it possible to externally control webrecorder-desktop to navigate to a page?

I'm using webrecorder-desktop because the in-browser version of webrecorder (either via webrecorder.io or `APP_HOST`) doesn't work with the SSO system used by the site I'm scraping.

I'm ideally hoping for an `APP_HOST` endpoint I can fire a POST request at with a URL as a payload, to trigger webrecorder-desktop to navigate to that URL.

**Comment by ikreymer** [comment]
Yes, that's a great idea and can allow folks to further extend the functionality / automation of the desktop app. 

 it should be possible to open the remote debugging port, and that will allow you to control the webview that is doing the recording. We can add that in next update.

To start the recording though, will require a separate puppeteer connections, eg. to enter the URL and click start. We can also look into making that simpler, through a simplified api.

There are also further updates coming that should aid in automating capture via the desktop app..
